modules/metastore/modules/metastore_search/tests/src/Functional/SearchTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace Drupal\Tests\metastore_search\Functional;
+
+use Drupal\metastore_search\Commands\RebuildTrackerCommands;
+use Drupal\metastore_search\Controller\SearchController;
+use GuzzleHttp\Client;
+use weitzman\DrupalTestTraits\ExistingSiteBase;
+
+/**
+ * Class SearchTest.
+ *
+ * @package Drupal\Tests\metastore_search\Functional
+ * @group metastore_search
+ */
+class SearchTest extends ExistingSiteBase {
+
+  /**
+   *
+   */
+  public function testDushCommands() {
+    $output = NULL;
+    $return = NULL;
+    exec('drush dkan:metastore-search:rebuild-tracker', $output, $return);
+    $this->assertEquals(0, $return);
```

Comment:
Why is there code after this non-expectException assertion?

Reply:
@thierrydallacroce did you read exec() as expectException() ?